scripts/anvil_tools/publish_workspaces_to_data_library.py

```diff
@@ -0,0 +1,128 @@
+"""Put/update Dataset attributes to workspaces parsed from input tsv file.
+
+Usage:
+    > python3 post_workspace_attributes.py -t TSV_FILE [-p BILLING-PROJECT] """
+
+import argparse
+import datetime
+import pandas as pd
+import requests
+
+from oauth2client.client import GoogleCredentials
+
+
+def get_access_token():
```

Comment:
if this and other functions (like write_output_report) are used in multiple scripts, you should make a common scripts file and import from there so you don't have duplicate code